docs/source/api/spatial-functions.rst

```diff
@@ -1777,6 +1777,49 @@ st_zmax
     :type col: Column
     :rtype: Column: DoubleType
 
+    :example:
+
+.. tabs::
+    .. code-tab:: py
+
+     >>> df = spark.createDataFrame([{'wkt': 'POINT (30 10 20)'}])
+     >>> df.select(st_zmax('wkt')).show()
+     +-----------------+
+     |st_minmaxxyz(wkt)|
+     +-----------------+
```

Comment:
@milos-colic wouldn't you just want this to be st_zmax(wkt) for the result column name?